ethereum/api/src/main/java/org/hyperledger/besu/ethereum/api/jsonrpc/internal/methods/TxPoolBesuPendingTransactions.java

```diff
@@ -49,7 +50,10 @@ public String getName() {
   @Override
   public JsonRpcResponse response(final JsonRpcRequestContext requestContext) {
 
-    final Integer limit = requestContext.getRequiredParameter(0, Integer.class);
+    final Collection<PendingTransaction> pendingTransactions =
```

Comment:
to avoid fetching twice all the txs in the pool, that could be a costly operation, reuse `pendingTransactions` below at line 64